src/apps/persistence/app_metadata.clj

```diff
@@ -405,6 +405,12 @@
    (delete app_references (where {:app_id app-id}))
    (dorun (map (partial add-app-reference app-id) references))))
 
+(defn set-htcondor-extra
+  [app-id extra-requirements]
+  (transaction
+    (delete apps_htcondor_extra (where {:apps_id app-id}))
+    (insert apps_htcondor_extra (values {:apps_id app-id, :extra_requirements extra-requirements}))))
```

Comment:
I'm not sure if it matters, but I could make it so this `insert` is only done if the extra-requirements are nonempty? The column on the table is only `NOT NULL`, however, and it won't hurt anything to have empty strings in there, so, shrug

Reply:
I think it should work now to pass that, or to pass an empty string (or, for good measure, a string that's only spaces).